tgui/packages/tgui/interfaces/Manufacturer/index.tsx

```diff
@@ -290,7 +293,7 @@ export const Manufacturer = (_, context) => {
                       </Button>
                     </Stack.Item>
                   </Stack>
-                  {data.queue.length > 0 && (
+                  {(data.queue && data.queue.length > 0) && (
```

Comment:
why not use ?. here and everywhere above

Reply:
Ah, probably forgot those. one moment, apologies